docs/rules/object-shorthand.md

```diff
@@ -98,6 +98,37 @@ You can set the option in configuration like this:
 }
 ```
 
+While set to `"always"`, `"methods"`, or `"properties"`, shorthand syntax using string literal keys can be ignored (for both properties and methods) using the optional parameter `"avoidQuotes"`. Note: The first parameter must be specified when using this optional parameter.
```

Comment:
You should say that `avoidQuotes` will not check property keys that are quoted and will only warn on unquoted keys that can be converted into shorthand without adding quotes.